examples/cfme.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: oVirt CFME deployment
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars_files:
+    # Contains encrypted `engine_password` and `metrics_password`
+    # varibale using ansible-vault
+    - passwords.yml
+
+  vars:
+    engine_fqdn: ovirt-engine.example.com
+    engine_user: admin@internal
+    engine_password: 123456
```

Comment:
Why not in vault? We should advertise and promote best practices (especially downstream). I see it up with the inclusion of 'passwords.yml' ?